Modules/Loadable/Data/qSlicerDataModule.cxx

```diff
@@ -71,15 +71,15 @@ QIcon qSlicerDataModule::icon()const
 //-----------------------------------------------------------------------------
 QStringList qSlicerDataModule::categories() const
 {
-  return QStringList() << "" << "Informatics";
+  return QStringList() << "" << tr("Informatics");
 }
 
 //-----------------------------------------------------------------------------
 QStringList qSlicerDataModule::dependencies() const
 {
   QStringList moduleDependencies;
   // Cameras: Required in qSlicerSceneReader
-  moduleDependencies << "Cameras";
+  moduleDependencies << tr("Cameras");
```

Comment:
This refers to the module name (that is not translated), not to the module title (that is translated). To make it clear that this text must not be translated, it would be safer to mark it as non-translatable. I tried to search for convention on how to make text as non-translatable in C/C++ source code, but I did not find anything. Therefore, we should come up with our own convention.

What do you think about marking non-translatable text using a macro or comment?

Option A: Using comment.

```
someFunction("some string"); // I18N:NO_TR - optional comment describing why not to translate
```

Option B: defining a no-op macro

Define this macro somewhere (in CTK?):
```
#define NO_TR(x) x
```

Usage:

```
someFunction(NO_TR("some string")); // optional comment describing why not to translate
```
abc(NO_TR("some string"));

@pieper @jcfr @cpinter

Reply:
Thanks for looking for a convention for this.  I'm kind of surprised since I would have thought this would be fairly common.  I was kind of thinking the convention would be `no_tr()` so that it's visually clear that it's related to `tr()`

Reply:
More similarity to `tr()` would be nice, but `tr()` is a function. Would you make `no_tr()` a function, too?

Qt uses macros for some translation features (QT_TR_...), so using all-caps macros would not be unprecedented.

Reply:
Yes, in general I would want all macros to be in all caps, but this might be an exception I would be okay with.  On the other hand making `no_tr` a method instead of a macro would be somehow more consistent and might have some value in the future.  I don't think it would have any performance impact since using strings as identifiers is already an option you would only use when performance isn't critical.  

Good point about the name `noTr` but in this case `tr` doesn't follow Qt convention anyway since it would normally be `translatable` or similar.

Maybe instead we should have a macro like `TOKEN` or `CTK_TOKEN` to be even more explicit about the intent of the string so it never get a `tr` added.

In any case if you still like `NO_TR` that's also fine.

Reply:
I don't find any of these names much better than the others. Let's see if @jcfr and @cpinter have any preferences.

Reply:
Thanks @lassoan and @pieper for your feedbacks. The translation function will be removed from the string in question in the next commit.

Thanks also for the interesting discussion you have about the convention we should use for marking non-translatable strings. 

I think that the first option (using comments) is easier to integrate on existing codes. However, the second one has the advantage to better highlight non-translatable strings since developers may not really consider comments when reading source codes, unless they find it necessary to do so. 

While you're deciding on the best approach to use, I'll keep marking translatable strings on other modules. The necessary work will be done for the non-translatable ones whenever a marking method is retained.

Reply:
To note hold up your work with this discussion, let's just use a comment now:

    someFunction(/*no tr*/"some string");

Once we have a decision, we can replace all these comments by the final solution.

Reply:
Alright, I'm fine with that. Thanks.

Reply:
Sorry just seeing this. I think `no_tr` would be most consistent with what we see in other places if it's a function, and `NO_TR` if it's a macro. I'm also fine with the comment approach as I see it in the commit right now, as it's pretty obvious (I don't see it easily ignorable since it's in-line, just before the string).